app/src/androidTest/java/co/condorlabs/emerald/emeraldgroup/EmeraldGroupButtonScreenExampleTest.kt

```diff
@@ -0,0 +1,97 @@
+package co.condorlabs.emerald.emeraldgroup
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.test.ExperimentalTestApi
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.assertIsNotEnabled
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import androidx.compose.ui.test.onNodeWithText
+import androidx.compose.ui.test.performClick
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import co.condorlabs.emerald.BaseTestingActivity
+import co.condorlabs.emerald.components.radiogroup.EmeraldRadioButtonState
+import co.condorlabs.emerald.components.radiogroup.EmeraldRadioGroup
+import co.condorlabs.emerald.theme.EmeraldTheme
+import org.junit.Assert
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@ExperimentalTestApi
+@RunWith(AndroidJUnit4::class)
+class EmeraldGroupButtonScreenExampleTest {
+
+    @get:Rule
+    val composeRule = createAndroidComposeRule<BaseTestingActivity>()
+
+    @Test
+    fun givenItemsWhenRadioGroupIsPressedThenShouldChangeTheValueOfTheItemYES() {
```

Comment:
remember we have a criteria for UI tests, please add Page Object Pattern

Reply:
Done